### DISCUSSION TITLE
Search templates

### DISCUSSION THREAD

**Discussion by dootyfree** [google/timesketch#1881] [Q&A, quick solutions, support]
Hello, 

Just wondering if it is expected to not see any search templates.  I do not have a drop down under explore search bar as mentioned in documentation.  Also attempted to import new templates, though they do not show up in gui as well.  Where can I find my search templates?

Best Regards,

**Comment by jaegeral** [comment]
Hm can you elaborate how you import the search templates? They should e.g. be created with analyzers.
